modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/logical/SqlScriptRunner.java

```diff
@@ -659,6 +659,8 @@ private void checkResultTuples(List<List<?>> res) {
 
         /** */
         private void checkEquals(String msg, String expectedStr, Object actual) {
+            expectedStr = F.isEmpty(expectedStr) ? expectedStr : expectedStr.trim();
```

Comment:
Why do we need it in the runner? It's better to fix tests if some tests are failed because of spaces. Looks like `test_trim.test` failed after this fix.